pom.xml

```diff
@@ -213,6 +225,7 @@
                     <show>public</show>
                     <skippedModules>benchmarks,examples,integration-tests,integration_tests,,simpleclient,simpleclient_bom,simpleclient_caffeine,simpleclient_common,simpleclient_dropwizard,simpleclient_graphite_bridge,simpleclient_guava,simpleclient_hibernate,simpleclient_hotspot,simpleclient_httpserver,simpleclient_jetty,simpleclient_jetty_jdk8,simpleclient_log4j,simpleclient_log4j2,simpleclient_logback,simpleclient_pushgateway,simpleclient_servlet,simpleclient_servlet_common,simpleclient_servlet_jakarta,simpleclient_spring_boot,simpleclient_spring_web,simpleclient_tracer,simpleclient_vertx,simpleclient_vertx4</skippedModules>
                     <excludePackageNames>io.prometheus.metrics.expositionformats.generated.*</excludePackageNames>
+                    <sourcepath>src/main/java;src/main/generated</sourcepath>
```

Comment:
This line has been added here. Is that on purpose? Seems unrelated to JUnit.

Reply:
This was needed to get `./mvnw javadoc:jar` to work.  Probably because I updated the version of `maven-javadoc-plugin`.

I have tried to compare the generated javadocs and they look substantially the same, with only minor differences in how the navigation works.  I see no differnces in the content.

To check, I cloned the repo to `client_java_copy` and did `./mvnw javadoc:jar` in both dirs to generate the javadocs.
Then I did `for file in **/target/apidocs; do diff "$file" "../client_java_copy/${file}" >> diff.txt; done` to capture the diffs to 
[diff.txt](https://github.com/prometheus/client_java/files/13754454/diff.txt)

Reply:
@fstab Is there anything more you you need for this?

Reply:
Sorry for dropping the ball here, I put it back on my TODO list and will get back to you soon :crossed_fingers:.

Reply:
@fstab any progress?